### PR TITLE
Updated .gitignore to prevent future accidental preview inclusions

### PR DESCRIPTION
**Pull request by bergerhoffer**
Also related to #251, updated .gitignore so that I don't accidentally include other preview files when I rename them to share the previews.

Had to do separately from #257 otherwise it wouldn't have let me remove that file.

### CODE REVIEW

**Review by bredamc — APPROVED** [review]
LGTM :)

**Comment by bergerhoffer** [comment]
Thanks! Merging before waiting for 3 approvals since this is just maintenance misc.

**Pull request merged**
